src/components/Property/PropertyDetails.js

```diff
@@ -2,52 +2,64 @@ import PropTypes from 'prop-types'
 import LocationAlerts from './LocationAlerts'
 import PersonAlerts from './PersonAlerts'
 import Tenure from './Tenure'
+import RaiseRepairLink from './RaiseRepairLink'
+import RepairUnavailableText from './RepairUnavailableText'
 
 const PropertyDetails = ({
   address,
   hierarchyType,
   locationAlerts,
   personAlerts,
   tenure,
-}) => (
-  <div>
-    <h1 className="govuk-heading-l">
-      {hierarchyType.subTypeDescription}: {address.addressLine}
-    </h1>
-
-    <div className="govuk-grid-row">
-      <div className="govuk-grid-column-one-half">
-        <div className="govuk-body-s">
-          <div className="property-details-main-section">
-            <span className="govuk-body-xs">Property details</span>
-            <br></br>
-            <span className="govuk-!-font-weight-bold text-green">
-              {address.addressLine}
-            </span>
-            <br></br>
-            {address.streetSuffix && (
-              <>
-                <span className="govuk-!-font-weight-bold text-green">
-                  {address.streetSuffix}
-                </span>
-                <br></br>
-              </>
-            )}
-            <span className="govuk-body-xs text-green">
-              {address.postalCode}
-            </span>
+}) => {
+  let labelToShow = (canRaiseRepair, subTypeDescription) => {
+    if (canRaiseRepair) {
+      return <RaiseRepairLink description={subTypeDescription} />
+    } else {
+      return <RepairUnavailableText />
+    }
+  }
+  return (
+    <div>
+      <h1 className="govuk-heading-l repair-laber">
+        {hierarchyType.subTypeDescription}: {address.addressLine}
+      </h1>
+      <div>
+        {labelToShow(tenure.canRaiseRepair, hierarchyType.subTypeDescription)}
```

Comment:
The components you've added look great.

This is nitpicky of me but please could we add lines 16-20 in here directly? It's a small change but I think in this case it should add a bit more readability. Breaking things down into small functions works really nicely in lots of places but here we're not getting much benefit from that function IMO. It's hard to accurately name what the function does which I think is nudging us in a direction - I know what you mean about 'label' but really it's a kind of 'linkOrText' function, and if we're at that stage I feel like just having the conditional in here straight up is better.

Reply:
As you explained in Slack, it sounds like I'm misunderstanding parts of how React works here with conditional rendering. I'll resolve and maybe we can revisit another time once we learn more - if applicable. Thank you for pointing that out.